bitnami/mariadb-galera/templates/_helpers.tpl

```diff
@@ -147,5 +147,65 @@ imagePullSecrets:
 {{- if and (contains "bitnami/" .Values.image.repository) (not (.Values.image.tag | toString | regexFind "-r\\d+$|sha256:")) }}
 WARNING: Rolling tag detected ({{ .Values.image.repository }}:{{ .Values.image.tag }}), please note that it is strongly recommended to avoid using rolling tags in a production environment.
 +info https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/
-{{- end }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Compile all warnings into a single message, and call fail.
+*/}}
+{{- define "mariadb-galera.validateValues" -}}
+{{- $messages := list -}}
+{{- $messages := append $messages (include "mariadb-galera.validateValues.rootPassword" .) -}}
+{{- $messages := append $messages (include "mariadb-galera.validateValues.password" .) -}}
+{{- $messages := append $messages (include "mariadb-galera.validateValues.mariadbBackupPassword" .) -}}
+{{- $messages := append $messages (include "mariadb-galera.validateValues.ldap" .) -}}
+{{- $messages := without $messages "" -}}
+{{- $message := join "\n" $messages -}}
+
+{{- if $message -}}
+{{-   printf "\nVALUES VALIDATION:\n%s" $message | fail -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Validate values of MariaDB Galera - must provide passwords when forced */}}
+{{- define "mariadb-galera.validateValues.rootPassword" -}}
+{{- if and .Values.rootUser.forcePassword (empty .Values.rootUser.password) -}}
+mariadb-galera: rootUser.password
+    A MariaDB Database Root Password is required (`rootUser.forcePassword=true` is set)
```

Comment:
If these errors are meant to be shown in a CLI environment, we should not need to escape code with "`".

Reply:
Applying this suggestion